### PR TITLE
feat: RegistryAPI - retrieve apps by discoveryMetadata

### PR DESCRIPTION
**Pull request by nightnei**

### CODE REVIEW

File: doc/registry_api.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Registry Api
```
Comment (StyleT) [inline doc/registry_api.md:1]: Add this doc here https://github.com/namecheap/ilc-sdk/blob/master/typedoc.json#L9

File: src/server/RegistryApi.ts [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    discoveryMetadata: DiscoveryMetadata;
+}[];
+
+type Options = {
+    logger?: Console;
+    maxAgeMillisecondsOfGetApps?: number;
+};
+
+/**
+ * Class RegistryApi provides capabilities to work with ILC Registry, e.g. retrieve some data etc.
```
Comment (StyleT) [inline src/server/RegistryApi.ts:18]:
```suggestion
 * Provides capability to work with ILC Registry to ILC apps or their backend services, e.g. retrieve some data etc.
```

**Review by StyleT — CHANGES_REQUESTED** [review]

File: src/server/RegistryApi.ts (hunk 2 above)
Comment (nightnei) [reply src/server/RegistryApi.ts]: done

File: doc/registry_api.md (hunk 1 above)
Comment (nightnei) [reply doc/registry_api.md:1]: done

**Review by StyleT — APPROVED** [review]

**Pull request merged**